### PR TITLE
[WiP] Solid OIDC tests

### PR DESCRIPTION
**Pull request by michielbdejong**
Companion to https://github.com/pdsinterop/php-solid-auth/pull/29
Working on issues found with https://inrupt.github.io/solid-oidc-tests/

### CODE REVIEW

**Comment by michielbdejong** [comment]
We should at least improve the error message to say 'Client ... is not registered'.

File: solid/lib/WellKnown/OpenIdConfigurationHandler.php [hunk 1]
```diff
@@ -117,6 +117,7 @@ private function respond($response) {
 			}
 		}
 		$result->setStatus($statusCode);
+		$result->addHeader('Access-Control-Allow-Origin', '*');
```
Comment (Potherca) [inline solid/lib/WellKnown/OpenIdConfigurationHandler.php:120]: I'm not sure about the CORS headers, maybe @ylebre has some opinions?

Comment (ylebre) [reply solid/lib/WellKnown/OpenIdConfigurationHandler.php:120]: This is fine - the rest of the system makes use of the middleware, but the wellknown doesn't so this needs to be in here.

**Comment by michielbdejong** [comment]
Closing this for now, follow-up in #104

**Pull request closed without merging**